client/my-sites/upgrades/checkout-thank-you/index.jsx

```diff
@@ -199,9 +216,7 @@ const CheckoutThankYou = React.createClass( {
 					primaryPurchase={ primaryPurchase }
 					selectedSite={ this.props.selectedSite } />
 
-				<div className="checkout-thank-you__features-header">
-					{ this.translate( "Get started with your site's new features" ) }
-				</div>
+				{ ! this.shouldHideFeaturesHeading() && <div className={ featuresHeaderClasses }>{ this.translate( 'What now?' ) }</div> }
```

Comment:
Would you mind explaining the rationale for this? This is a single line of text with a relatively simple display condition, and I don't think moving it into its own component makes it easier to understand how `CheckoutThankYou` is rendered (for me, at least).